dev_utils/config.yaml

```diff
@@ -46,6 +47,7 @@ broker:
 c4gh:
   passphrase: "oaagCP1YgAZeEyl2eJAkHv9lkcWXWFgm"
   filepath: "./dev_utils/c4gh.sec.pem"
+  newpublickey: "./dev_utils/c4gh-new.pub.pem"
```

Comment:
A better name for this should be though of, that is more descriptive other than just  `new`.

examples: 
* syncPubKey 
* backupPubKey